deploy/packer/base.json

```diff
@@ -1,12 +1,41 @@
 {
   "description": "taskcluster worker system dependencies",
   "variables": {
-    "workerRevision": ""
+    "papertrail":             "",
+    "privateKeyLocation":     "",
+    "sslCertificateLocation": "",
+    "sslKeyLocation":         "",
+    "workerRevision":         ""
   },
   "provisioners": [
+    { "type": "file",
+      "source": "{{user `privateKeyLocation`}}",
+      "destination": "/tmp/docker-worker-priv.pem"
+    },
+    { "type": "file",
+      "source": "{{user `sslCertificateLocation`}}",
+      "destination": "/tmp/star_taskcluster-worker_net.crt"
+    },
+    { "type": "file",
+      "source": "{{user `sslKeyLocation`}}",
+      "destination": "/tmp/star_taskcluster-worker_net.key"
```

Comment:
why not call it `worker-ssl.key`... if someday TC is deployed outside tc.net context it would be weird...

Anyways, not you don't have to fix it :)